prboom2/src/gl_sky.c

```diff
@@ -737,7 +737,9 @@ static void gld_BuildSky(int row_count, int col_count, SkyBoxParams_t *sky, int
 static void RenderDome(SkyBoxParams_t *sky)
 {
   int i, j;
+#ifdef USE_VBO
```

Comment:
This variable is unused due to a later `#ifdef USE_VBO`.  We never define `USE_VBO` in any build as far as I can tell, so a lot of VBO-related code is simply turned off.  Is it supposed to be that way?

Reply:
Pr+ has a `// #define USE_VBO` line, so I assume this was only tested out but never finished. Could you make a separate PR that just removes the remaining stuff related to it? Or - if you think the stuff there is important, let me know if we should do something with it 😄

Reply:
It does save on per-frame memory transfer costs, so it's probably worthwhile to complete the implementation.